### ISSUE TITLE
PlugInstall Crashes on Vim 8.0

### ISSUE THREAD

**Issue by cmather** [junegunn/vim-plug#523]
PlugInstall is crashing on Vim version 8.0. See error message in second screenshot below. Vim exits with error message printed to the terminal.

![screen shot 2016-09-15 at 3 10 31 pm](https://cloud.githubusercontent.com/assets/262179/18569320/c045b936-7b56-11e6-938f-a66937246edb.png)

![screen shot 2016-09-15 at 3 10 41 pm](https://cloud.githubusercontent.com/assets/262179/18569325/c21a4bb4-7b56-11e6-8aad-5698e4c25b27.png)

```
8.0
```

- Type:
    - [X] Bug
    - [ ] Enhancement
    - [ ] Feature Request
    - [ ] Question
- OS:
    - [ ] All/Other
    - [ ] Linux
    - [X] OS X
    - [ ] Windows
- Vim:
    - [X] Terminal Vim
    - [ ] GVim
    - [ ] Neovim

**Comment by junegunn** [comment]
Do you use RVM? Please see if this link helps: https://github.com/junegunn/vim-plug/wiki/faq#vim-caught-deadly-signal-segv

**Comment by benoittgt** [comment]
I was getting the same issue and the link that Junegunn gave fixed the issue. Thanks !

**Comment by cmather** [comment]
Oh interesting thanks for the tip @junegunn. I use rbenv with the 2.3.1 global version of ruby. I compiled vim by hand with the enable-rubyinterp config option but maybe I also need to provide a path to the correct ruby install. I'll report back my findings.

**Comment by cmather** [comment]
I was able to fix this but I'm not entirely clear what happened. I'll reproduce the fix below for anyone it might help. I started off with Ruby 2.3.1 set with `rbenv global 2.3.1` and then compiled vim with the `--enable-rubyinterp` flag. I still had issues even though I noticed with `vim --version` that the compiled version was pointing to the correct ruby folder. When I recompiled vim using the system version of ruby it seems to work.

To fix I did this:

```bash
> rbenv shell system
> ruby --version //=> ruby 2.0.0p648
> cd ~/src/vim
> ./configure --enable-rubyinterp
> make && sudo make install
```

So maybe it's a Ruby 2.3.1 issue or perhaps more likely it's a conflict between vim being compiled with version 2.3.1. and trying to use the system version.
